includes/admin/class-wc-stripe-admin-notices.php

```
@@ -476,38 +476,49 @@ public function subscription_check_detachment() {
-			);
-			$customer_stripe_page = sprintf(
-				'<a href="%s">%s</a>',
-				esc_url( WC_Stripe_Subscriptions_Helper::STRIPE_CUSTOMER_PAGE_BASE_URL . WC_Stripe_Order_Helper::get_instance()->get_stripe_customer_id( $subscription ) ),
-				esc_html(
-					/* translators: this is a text for a link pointing to the customer's page on Stripe */
-					__( 'Stripe customer page &rarr;', 'woocommerce-gateway-stripe' )
-				)
-			);
+		// If not detached but the user dismissed the notice prior, clear the meta so it can show if later detached.
+		$detachment_meta_key = '_wc_stripe_subscription_detached_notice_dismissed';
+		if ( ! WC_Stripe_Subscriptions_Helper::is_subscription_payment_method_detached( $subscription ) ) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Subscriptions_Helper' in method 'subscription_check_detachment'. (undefined)

(StaticAccess)